crates/lib/src/install/config.rs

```diff
@@ -191,11 +192,21 @@ impl InstallConfiguration {
 #[context("Loading configuration")]
 /// Load the install configuration, merging all found configuration files.
 pub(crate) fn load_config() -> Result<Option<InstallConfiguration>> {
+    load_config_at("/")
+}
+
+pub(crate) fn load_config_at<P: AsRef<Path>>(root_dir: P) -> Result<Option<InstallConfiguration>> {
     let env = EnvProperties {
         sys_arch: std::env::consts::ARCH.to_string(),
     };
-    const SYSTEMD_CONVENTIONAL_BASES: &[&str] = &["/usr/lib", "/usr/local/lib", "/etc", "/run"];
-    let fragments = liboverdrop::scan(SYSTEMD_CONVENTIONAL_BASES, "bootc/install", &["toml"], true);
+    let root_dir = root_dir.as_ref();
+    let systemd_conventional_bases = &[
+        root_dir.join("usr/lib"),
+        root_dir.join("usr/local/lib"),
+        root_dir.join("etc"),
+        root_dir.join("run"),
+    ];
```

Comment:
How about keeping the `const SYSTEMD_CONVENTIONAL_BASES` above but drop the leading `/` and then `let systemd_conventional_bases = SYSTEMD_CONVENTIONAL_BASES.map(|v| root_dir.join(v)).collect::<Vec<_>>();` or so